app/views/payment_tracking/index.html.erb

```diff
@@ -48,6 +49,7 @@
           <td><%= if rental.renter.department then rental.renter.department.name else rental.renter.full_name end %></td>
           <td><%= ((Time.current-rental.created_at) / 1.days ).floor %></td>
           <td><%= number_to_currency rental.balance %></td>
+          <td><%= rental.payments.map { |ft| ft.transactable.reference }.join ', ' %></td>
```

Comment:
This is safe if there are no payments, it will return a empty `ActiveRecord::Relation`, performing a map on that object will yield an empty array joined by , is just nil.